### PR TITLE
Added link for @bors cheatsheet to Helpful Links page.

### PR DESCRIPTION
**Pull request by alexispurslane**

### CODE REVIEW

**Comment by christopherdumas** [comment]
(Squashed the commits, too)

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -225,6 +225,7 @@ are:
 * Although out of date, [Tom Lee's great blog article][tlgba] is very helpful
 * [rustaceans.org][ro] is helpful, but mostly dedicated to IRC
 * The [Rust Compiler Testing Docs][rctd]
+* For @bors, [this cheetsheat][cheetsheat] is helpful (Remember to replace `@homu` with `@bors` in the commands that you use.)
```
Comment (bluss) [inline CONTRIBUTING.md]: Nice link to add! I noted a typo though, should be *cheatsheet*.

Comment (steveklabnik) [inline CONTRIBUTING.md]: Ahh yes, this is true. @christopherdumas mind sending a follow-up PR with the fix?

**Pull request merged**